workflows/cloudify_system_workflows/snapshots/snapshot_restore.py

```diff
@@ -352,6 +355,62 @@ def _restore_deployment_envs(self, postgres):
 
         self._log_final_information(deps_with_failed_plugins)
 
+    def _restore_inter_deployment_dependencies(self):
+        if (self._snapshot_version < V_4_6_0 or
+                self._snapshot_version > V_5_0_5):
+            return
```

Comment:
this could use a comment as to _why_ this check is here ("because managers older than 4.6.0 didn't support components, and managers newer than 5.0.5 have the dependencies as part of the database dump")

Reply:
Added